src/scripts/main.js

```diff
@@ -1,3 +1,242 @@
 'use strict';
 
-// write your code here
+const fieldRows = document.querySelector('tbody').rows;
+const startGame = document.querySelector('.start');
+const messageLose = document.querySelector('.message-lose');
+const messageWin = document.querySelector('.message-win');
+const messageStart = document.querySelector('.message-start');
+const gameScore = document.querySelector('.game-score');
+
+let score = 0;
+let rows = 4;
+let columns = 4;
+let field = [
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+  [0, 0, 0, 0]
+];
+
+function gameStart() {
+  field = [
+    [0, 0, 0, 0],
+    [0, 0, 0, 0],
+    [0, 0, 0, 0],
+    [0, 0, 0, 0]
+  ];
+
+  addTwoDeuces();
+  addTwoDeuces();
+  updateHtml();
+};
+
+startGame.addEventListener('click', () => {
+  startGame.classList.replace('start', 'restart');
+  startGame.innerHTML = 'Restart';
+  messageStart.classList.add('hidden');
+  messageLose.classList.add('hidden');
+  messageWin.classList.add('hidden');
+  gameScore.innerHTML = 0;
+
+  gameStart();
+})
+
+document.addEventListener('keyup', (event) => {
+  if (!startGame.classList.contains('restart')) {
```

Comment:
Add `e.preventDefault()` before you handle the event. Because I have page scroll if I press up/down buttons.